shared/constants/network.ts

```diff
@@ -612,6 +611,10 @@ export const BUYABLE_CHAINS_MAP: {
     nativeCurrency: CURRENCY_SYMBOLS.PALM,
     network: 'palm',
   },
+  [CHAIN_IDS.LINEA_MAINNET]: {
+    nativeCurrency: CURRENCY_SYMBOLS.ETH,
```

Comment:
I was expecting as an L2 that we use ETH just like Optimism does, can we confirm this with linea team?

Reply:
Wouldn't that make the native ticker be OP? 🤔 this is currently `ETH` even in their block tracker since `OP` is an ERC-20
![image](https://github.com/MetaMask/metamask-extension/assets/1024246/fd65d0a8-67f0-4279-9735-a2ec2b0f414a)